pkgs/development/python-modules/pylibftdi/default.nix

```
@@ -0,0 +1,40 @@
+{ stdenv
+, pkgs
```

you should avoid importing pkgs, instead pass the packages you need individually, look at other comment
```suggestion
, libusb1
```